tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss

```diff
@@ -101,9 +101,16 @@ mat-slider {
   align-items: center;
   display: flex;
   cursor: pointer;
+  font-size: inherit;
+  height: 25px;
+  padding-left: 5px;
+  padding-right: 8px;
+  &.toggle-opened {
+    background-color: mat.get-color-from-palette(mat.$grey-palette, 800);
```

Comment:
Note that this doesn't work well in light mode.

Reply:
Oops. Fixed that up.